homework/02LessonTwo.js

```diff
@@ -53,22 +79,25 @@ function lessThanOrEqualTo(parameter1, parameter2) {
 //
 // What is a function? How do you define a function in Javascript?
 //
-// Your Answer Goes Here:
-//
+// Your Answer Goes Here: A function is used to help solve any problems in programming
+// A JavaScript function is a block of code designed to perform a particular task.
+// defining a function by assigning it variable and calling it.
```

Comment:
Good job. However, there is a distinction between defining a function and calling a function.

[MDN Functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Functions)